bindings/gumjs/gumdukcore.c

```diff
@@ -145,6 +146,125 @@ union _GumFFIValue
   gint64 v_sint64;
   guint64 v_uint64;
 };
+#else
+/*
+ * On little endian the low order bytes of a value appear at the lowest address
+ * in memory. To the left in the diagram below. Thus it is trivial and transparent 
```

Comment:
Would be good to ensure no lines exceed 80 cols. (I'll let you catch the other places to avoid noisy review feedback.)